archive_viewer/av_file_convert.py

```diff
@@ -0,0 +1,374 @@
+import json
+import logging
+import xml.etree.ElementTree as ET
+from re import compile
+from os import (path, getenv)
+from typing import (Dict, Union)
+from pathlib import Path
+from datetime import datetime
+from argparse import (ArgumentParser, Action, Namespace)
+from qtpy.QtGui import QColor
+from collections import OrderedDict
+from pydm.widgets.timeplot import PyDMTimePlot
+
+
+if __name__ in logging.Logger.manager.loggerDict:
+    logger = logging.getLogger(__name__)
+else:
+    logger = logging.getLogger("")
+    handler = logging.StreamHandler()
+    formatter = logging.Formatter("[%(asctime)s] [%(levelname)-8s] - %(message)s")
+    handler.setFormatter(formatter)
+    logger.addHandler(handler)
+    logger.setLevel("INFO")
+    handler.setLevel("INFO")
+
+
+class ArchiveViewerFileConverter():
```

Comment:
would it be possible to add a class description? thanks

Reply:
Solved in commit 3f00fff13322325cd1f81101ea9b124ff61f02fb